falcosidekick/values.yaml

```diff
@@ -57,10 +57,34 @@ config:
   customfields: ""
   # -- a list of escaped comma separated Go templated fields to add to falco events, syntax is "key:template\,key:template"
   templatedfields: ""
-
-  # -- folder which will used to store client.crt, client.key and ca.crt files for mutual tls (default: "/etc/certs")
+  # -- if not empty, the brackets in keys of Output Fields are replaced
+  bracketreplacer: ""
+  # -- folder which will used to store client.crt, client.key and ca.crt files for mutual tls for outputs, will be deprecated in the future (default: "/etc/certs")
   mutualtlsfilespath: "/etc/certs"
 
+  mutualtlsclient:
+    # -- client certification file for mutual TLS client certification, takes priority over mutualtlsfilespath if not empty
+    certfile: ""
+    # -- client key file for mutual TLS client certification, takes priority over mutualtlsfilespath if not empty
+    keyfile: ""
+    # -- CA certification file for server certification for mutual TLS authentication, takes priority over mutualtlsfilespath if not empty
+    cacertfile: ""
+
+  tlsserver:
+    # -- if true TLS server will be deployed instead of HTTP
+    deploy: false
+    # -- server certification file for TLS Server
+    certfile: "/etc/certs/server/server.crt"
+    # -- server key file for TLS Server
+    keyfile: "/etc/certs/server/server.key"
+    # -- if true mutual TLS server will be deployed instead of TLS, deploy also has to be true
+    mutualtls: false
+    # --  CA certification file for client certification if mutualtls is true
+    cacertfile: "/etc/certs/server/ca.crt"
+    # -- port to serve http server serving selected endpoints
+    notlsport: 2810
```

Comment:
we need to open this port in the service and ingress with the excluded paths from TLS